server/src/uuid/model/taxonomy_term/mod.rs

```diff
@@ -650,4 +650,57 @@ impl TaxonomyTerm {
 
         Ok(())
     }
+
+    pub async fn delete_entity_link<'a, E>(
+        payload: &taxonomy_delete_entity_link_mutation::Payload,
+        executor: E,
+    ) -> Result<(), operation::Error>
+    where
+        E: Executor<'a>,
+    {
+        let mut transaction = executor.begin().await?;
+
+        let instance_id = Self::get_instance_id(payload.taxonomy_term_id, &mut transaction).await?;
+
+        for child_id in payload.entity_ids.clone() {
+            let term_taxonomy_entity_id = sqlx::query!(
+                r#"
+                    SELECT id FROM term_taxonomy_entity
+                        WHERE entity_id = ?
+                        AND term_taxonomy_id = ?
+                "#,
+                child_id,
+                payload.taxonomy_term_id
+            )
+            .fetch_optional(&mut transaction)
+            .await?
+            .ok_or(operation::Error::BadRequest {
+                reason: format!(
+                    "Id {} is not linked to taxonomy term {}",
+                    child_id, payload.taxonomy_term_id
+                ),
+            })?
+            .id as i32;
+
```

Comment:
Add check that `entity` has also another link to another taxonomy term (we do not want to have entties without a link to a taxonomy term)